plugin.xml

```diff
@@ -94,6 +94,14 @@ Fonctionnalités
          <compatibility>9.2.4</compatibility>
          <compatibility>9.3</compatibility>
       </version>
+      <version>
+         <num>2.0.0-rc.3</num>
+         <compatibility>9.2.1</compatibility>
+         <compatibility>9.2.2</compatibility>
+         <compatibility>9.2.3</compatibility>
+         <compatibility>9.2.4</compatibility>
+         <compatibility>9.3</compatibility>
```

Comment:
Are you not going to add others 9.3.x versions?

Reply:
the expression 9.3 includes all 9.3.x versions.

As the plugin uses a feature not available in 9.2.0, all revisions 9.2.1, 9.2.2, and so on are explicitly included.